doc/source/scipyoptdoc.py

```diff
@@ -80,7 +80,7 @@ def run(self):
             # Implementation function
             impl_name = self.options['impl']
             impl_obj = _import_object(impl_name)
-            impl_args, impl_varargs, impl_keywords, impl_defaults = getfullargspec_no_self(impl_obj)[:4]
+            impl_args, impl_varargs, impl_keywords, impl_defaults = getfullargspec_no_self(impl_obj)[:4]  # noqa: E501
```

Comment:
any nice way to split this or no?

Reply:
I would be happy to unpack two at a time on separate lines, or sometimes I'll store everything in a temporary tuple, then unpack the tuple.

Reply:
Turns out two of the variables are unused. Can we do this?
```suggestion
            impl_args, _, _, impl_defaults = getfullargspec_no_self(impl_obj)[:4]
```

Reply:
Sure.